tf2onnx/tfonnx.py

```diff
@@ -2475,16 +2475,26 @@ def topological_sort(g, continue_on_error):
             pass
 
 
-def run_late_rewriters(g, funcs, continue_on_error):
+def run_rewriters(g, funcs, continue_on_error):
+    """Rewrite the original graph and body graphs of nodes"""
+    # NOTE: not topologically sort graph here
```

Comment:
please add more words here about:
- rewriter is expected to sort nodes on its own
- graph may contains circle, current topo sort cannot handle it.
- why process parent graph first then child graphs.

Reply:
added some comments, the third question is obvious if one reads the code.